lib/rihanna.ex

```diff
@@ -85,6 +91,87 @@ defmodule Rihanna do
     raise ArgumentError, @enqueue_help_message
   end
 
+  @schedule_help_message """
+  Rihanna.schedule requires either two arguments in the form {mod, fun, args}
```

Comment:
No need to specify args here, since they are in the typespec.

Reply:
This is the help message shown as an `ArgumentError` when invalid arguments are provided to the `Rihanna.schedule` function.

Reply:
Doh, you are quite right.